### PR TITLE
PIM-5169: fix mass edit attribute selection

### PR DESCRIPTION
**Pull request by solivier**
| Q                 | A
| ----------------- | ---
| Specs             | n
| Behats            | n
| Blue CI           | Yes
| Changelog updated | yes
| Review and 2 GTM  | no

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig [hunk 1]
```diff
@@ -176,9 +176,12 @@
                         $saveButton.click();
                     });
 
+
```
Comment (solivier) [inline src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig]: to remove

File: CHANGELOG-1.4.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## Bug fixes
 - PIM-5163: Fix the VersionRepository on MongoDB to take the most recent entry for product resources
+- PIM-5169: Fix mass edit attribute selection wile using a small screen resolution
```
Comment (willy-ahva) [inline CHANGELOG-1.4.md]: while

**Comment by phaseinducer** [comment]
GTM :+1:

**Comment by sumbobyboys** [comment]
GTM

**Comment by jjanvier** [comment]
gtm

**Pull request merged**